cypress/support/fragments/orders/orderEditForm.js

```diff
@@ -32,6 +34,9 @@ export default {
   selectOrderTemplate(templateName) {
     this.selectDropDownValue('Template name', templateName);
   },
+  selectOrderType(type) {
```

Comment:
This action can be done by already implemented method: fillOrderInfoSectionFields({orderType})